### ISSUE TITLE
Create serialization tests

### ISSUE THREAD

**Issue by aloisdeniel** [dnb-asa/figma#12]
To validate type deserialization, local json responses should be added to tests and the parsing result should be checked.

**Comment by arnemolland** [comment]
Absolutely! I'll make sure to make the test project/files public as well.
